app/views/assistants/_assistant.html.erb

```diff
@@ -1,58 +1,29 @@
-<%# locals: (assistant:, settings: true, assistant_counter: -1) %>
-<% selected = assistant == @assistant %>
-<% first = assistant_counter == 0 %>
+<%# locals: (assistant:, settings: true, assistant_counter: -1) %> <% selected = assistant == @assistant %> <% first = assistant_counter == 0 %>
```

Comment:
Did you mean to pull all these onto one line?

Reply:
VS Code "prettified" things. Reverting now.